pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [tool.poetry]
 name = "terralab-cli"
 # version is auto-incremented. do not edit.
```

Comment:
no need to edit this yourself

Reply:
if only there was a comment saying not to edit this! 😭 😂